docs/book/v3/coding-style-guide.md

```diff
@@ -0,0 +1,1353 @@
+>
+> PHPDoc tags `@param`, `@return` and `@throws` SHOULD not be aligned or
+> contain multiple spaces between the tag, type and description.
+>
+> If a function throws any exceptions, it SHOULD be documented with
+> `@throws` tags.
+>
+> DocBlocks MUST follow this specific order of annotations with empty
+> newline between specific groups:
+>
+
+```php
+/**
+ * <Summary>
+ *
+ * <Description>
+ *
+ * @internal
+ * @deprecated
+ *
+ * @link
+ * @see
+ * @uses
+ *
+ * @param
+ * @return
+ * @throws
+ */
+```
+
+> The annotations `@api`, `@author`, `@category`, `@created`, `@package`,
+> `@subpackage` and `@version` MUST NOT be used in comments. Git commits
+> provide accurate information.
+>
+> The words _private_, _protected_, _static_, _constructor_, _deconstructor_,
+> _Created by_, _getter_ and _setter_, MUST NOT be used in comments.
+>
+> The `@var` tag MAY be used in inline comments to document the _Type_
+> of properties. Single-line property comments with a `@var` tag SHOULD
+> be written as one-liners. The `@var` MAY NOT be used for constants.
+>
+> The correct tag case of PHPDocs and PHPUnit tags MUST be used.
+>
+> Inline DocComments MAY be used at the end of the line, with at least a
+> single space preceding. Inline DocComments MUST NOT be placed after curly
+> brackets.
+>
+> Heredoc and nowdoc tags MUST be uppercase without spaces.
+
+[PSR-1]: https://www.php-fig.org/psr/psr-1/
+[PSR-12]: https://www.php-fig.org/psr/psr-12/
+[keywords]: https://www.php.net/manual/en/reserved.keywords.php
+[types]: https://www.php.net/manual/en/reserved.other-reserved-words.php
+[arithmetic]: https://www.php.net/manual/en/language.operators.arithmetic.php
+[assignment]: https://www.php.net/manual/en/language.operators.assignment.php
+[comparison]: https://www.php.net/manual/en/language.operators.comparison.php
+[bitwise]: https://www.php.net/manual/en/language.operators.bitwise.php
+[logical]: https://www.php.net/manual/en/language.operators.logical.php
+[string]: https://www.php.net/manual/en/language.operators.string.php
+[type]: https://www.php.net/manual/en/language.operators.type.php
+[short open tag]: https://www.php.net/manual/en/language.basic-syntax.phptags.php
+[70.deprecated]: https://www.php.net/manual/en/migration70.deprecated.php
+[71.deprecated]: https://www.php.net/manual/en/migration71.deprecated.php
+[72.deprecated]: https://www.php.net/manual/en/migration72.deprecated.php
+[73.deprecated]: https://www.php.net/manual/en/migration73.deprecated.php
+[74.deprecated]: https://www.php.net/manual/en/migration74.deprecated.php
+[backtick operator]: https://www.php.net/manual/en/language.operators.execution.php
+[goto]: https://www.php.net/manual/en/control-structures.goto.php
+[global]: https://www.php.net/manual/en/language.variables.scope.php#language.variables.scope.global
+[PHP_SAPI]: https://www.php.net/manual/en/function.php-sapi-name.php#refsect1-function.php-sapi-name-notes
+[aliases]: https://www.php.net/manual/en/aliases.php
```

Comment:
```suggestion
[keywords]: https://www.php.net/manual/reserved.keywords.php
[types]: https://www.php.net/manual/reserved.other-reserved-words.php
[arithmetic]: https://www.php.net/manual/language.operators.arithmetic.php
[assignment]: https://www.php.net/manual/language.operators.assignment.php
[comparison]: https://www.php.net/manual/language.operators.comparison.php
[bitwise]: https://www.php.net/manual/language.operators.bitwise.php
[logical]: https://www.php.net/manual/language.operators.logical.php
[string]: https://www.php.net/manual/language.operators.string.php
[type]: https://www.php.net/manual/language.operators.type.php
[short open tag]: https://www.php.net/manual/language.basic-syntax.phptags.php
[70.deprecated]: https://www.php.net/manual/migration70.deprecated.php
[71.deprecated]: https://www.php.net/manual/migration71.deprecated.php
[72.deprecated]: https://www.php.net/manual/migration72.deprecated.php
[73.deprecated]: https://www.php.net/manual/migration73.deprecated.php
[74.deprecated]: https://www.php.net/manual/migration74.deprecated.php
[backtick operator]: https://www.php.net/manual/language.operators.execution.php
[goto]: https://www.php.net/manual/control-structures.goto.php
[global]: https://www.php.net/manual/language.variables.scope.php#language.variables.scope.global
[PHP_SAPI]: https://www.php.net/manual/function.php-sapi-name.php#refsect1-function.php-sapi-name-notes
[aliases]: https://www.php.net/manual/aliases.php
```

Removes language from links so the PHP website can autodetect from the users browser